stock_orderpoint_generator/tests/test_orderpoint_generator.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# © 2016 Cyril Gaudin (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.exceptions import UserError
+from openerp.tests import TransactionCase
+
+
+class TestOrderpointGenerator(TransactionCase):
+
+    def setUp(self):
+        super(TestOrderpointGenerator, self).setUp()
+
+        self.wizard_model = self.env['stock.warehouse.orderpoint.generator']
+
+        self.orderpoint_model = self.env['stock.warehouse.orderpoint']
+
+        self.orderpoint_template_model = self.env[
+            'stock.warehouse.orderpoint.template'
+        ]
+
+        self.product_model = self.env['product.product']
+        self.p1 = self.product_model.create({'name': 'Unittest P1'})
+        self.p2 = self.product_model.create({'name': 'Unittest P2'})
+
+        self.assertEqual(0, self.orderpoint_model.search_count([
+            ('name', '=', 'OP/000445')
+        ]))
+
+        self.template = self.orderpoint_template_model.create({
+            'company_id': self.ref('base.main_company'),
+            'location_id': self.ref('stock.stock_location_stock'),
+            'name': 'OP/000445',
+            'product_max_qty': 15.0,
+            'product_min_qty': 5.0,
+            'qty_multiple': 1,
+            'warehouse_id': self.ref('stock.warehouse0')
+        })
+
+    def check_orderpoint(self):
+        orderpoints = self.orderpoint_model.search([
+            ('name', '=', 'OP/000445')
+        ])
+
+        orderpoints = sorted(orderpoints, key=lambda o: o.product_id.id)
```

Comment:
You can sort while doing the search using attribute `order`

    model.search(domain, order="product_id")